### DISCUSSION TITLE
"Shift" key to have two function for a button

### DISCUSSION THREAD

**Discussion by nicolasvair** [shanteacontrols/OpenDeck#221] [General]
Hi,

About the same project I'm working on, I'd like to have a key that act as "shift" when pressed to be able to map 2 functions to a button. An easy way to do it would be to switch preset with that key BUT I haven't been able to make it work as "latch" and come back to the first preset afterwards..

Any idea about how to do this ? 

Thank you !

**Comment by nicolasvair** [comment]
Also, I should add that I can't do it on my daw. I'm using ableton and the only option to do this would be a midi script in python... and i'd need to do it for all the mapping..

**Comment by paradajz** [comment]
That's currently not possible in the OpenDeck firmware as something like that would be fairly complex to implement.
